src/routes/send.tsx

```diff
@@ -21,7 +21,7 @@ export const action: ActionFunction = async ({ request }) => {
   const url = formData.get("url");
 
   if (!validateURL(url)) {
-    return json("Lütfen geçerli bir URL adresi girin.", { status: 400 })
+    return json("Lütfen geçerli bir URL adresi girin.", { status: 400 });
```

Comment:
It might be good idea to separate this constant string to another `class` or `file`, and use it from there, so migrating to some `internationalization` package can become easier.

Reply:
i didn't want to do that simply because this is for kamp.us community.